app/routes/__index/index.tsx

```diff
@@ -12,18 +12,14 @@ export const loader = async ({ request }: LoaderArgs) => {
 };
 
 export default function IndexRoute() {
-  const data = useLoaderData<typeof loader>();
-
+  const user = useUser();
   return (
     <div>
```

Comment:
Need to wrap it up in Chakra container, something like
```typescript
<Container maxW="container.xl">
```

Reply:
I'm not sure about touching index.tsx, because as far as I recall we won't have a home page according to one of our previous meeting

Reply:
I think this should be replaced by the domains page (as per https://github.com/Seneca-CDOT/starchart/issues/46)